tests/specs/benchmarks/verification.k

```diff
@@ -211,6 +211,10 @@ module VERIFICATION
 
     rule #ceil32(X) => X requires X modInt 32 ==Int 0
 
+    rule 0 <=Int #ceil32(X) -Int X => true requires 0 <=Int X [simplification]
+
+    rule 0 <Int 1 <<Int N => true requires 0 <=Int N [simplification]
+
     rule X <=Int #ceil32(X) => true
       requires X >=Int 0
```

Comment:
@ehildenb this one

Reply:
I'm trying with the other lemma removed and this one marked `simplification`, but I don't think it will work, because it's specifically to simplify out the form `0 <=Int #ceil32(X) -Int X`, not for simplifying out `X <=Int #ceil32(X)`. But if it works I'll push that up.